app/components/conceptAdd/conceptAdd.controller.js

```diff
@@ -114,6 +116,11 @@
 		}
 		function postConcept(){
 			vm.result = conceptsService.postConcept(vm.concept, vm.localizedConcepts);
+			if(angular.isDefined(vm.result)&&vm.result.success) $location.path('/concept');
+		}
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw '$location'.](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2339254595/issues/source?bid=3196969&fileBranchId=3209439#l123)